### ISSUE TITLE
WCAG : Asterisk is read for mandatory fields

### ISSUE THREAD

**Issue by Amit-Ag** [ngx-formly/ngx-formly#2887]
For creating accessible web applications; screen readers like NVDA should be able to correctly read out form field labels.

Dependencies:
"@ngx-formly/core" : "5.10.14"
"@ngx-formly/material" : "5.10.14"
 "@ngx-formly/core" : "5.10.14"
 "@angular/common": "~9.1.13",
 "@angular/compiler": "~9.1.13",
 "@angular/core": "~9.1.13",
 "@angular/forms": "~9.1.13",
"@angular/material": "^9.2.4",

Steps to replicate:
1. Go to https://stackblitz.com/angular/nlvpljvvrvd
2. Click on 'Open in New Window' button to open the page in new tab
3. On the new page use NVDA screen reader
4. Use tab to navigate to the input field
5. Listen to NVDA read the input field label

Observe: 
1. The field label is read as: '**Input *  edit  required  Placeholder  blank'**

Expectation:
1. The field label should be read as : '**Input edit  required  Placeholder  blank**'

Note: The asterisk is currently being read.

**RCA:**
https://github.com/ngx-formly/ngx-formly/blob/6f0d06083cc65916157db3840d8f03be225e3f77/src/ui/material/form-field/src/form-field.wrapper.ts#L60

**This line responsible for showing the required asterisk marker does not have aria-hidden="true".**

Reference to Angular material:
https://github.com/angular/components/blob/fec578dbccd8a4476cabce3623c4dc084a91ec6b/src/material/form-field/form-field.html#L54

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.10.22](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.10.22) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
